vital/bitflags.h

```diff
@@ -0,0 +1,107 @@
+// This file is part of ViViA, and is distributed under the
```

Comment:
I assume this should be updated from ViViA to KWIVER?

Reply:
Arf, yes. Good catch, thanks! Should be fixed.